packages/table/src/table.tsx

```diff
@@ -450,6 +450,18 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
+    public resizeRowsBasedOnColumnHeight(columnIndex: number) {
+        const {locator} = this.state;
+
+        const tallest = locator.getTallestVisibleCellInColumn(columnIndex);
+        const rowHeights = this.state.rowHeights.slice();
+        for (let row = 0; row < rowHeights.length; row++) {
+            rowHeights[row] = tallest;
+        }
```

Comment:
More succinct/declarative: `const rowHeights = Array(this.state.rowHeights.length).fill(tallest);`